client/dom/table.ts

```diff
@@ -143,8 +146,9 @@ export function renderTable({
 					if (noButtonCallback) for (const [i, node] of nodes.entries()) noButtonCallback(i, node)
 				})
 			checkboxH.node().checked = selectAll
+			if (!showHeader)
```

Comment:
Airen should evaluate if this change and the one at line 189 is okay. but i will merge now

Reply:
okay, the lollipop track seems fine to me. nothing seems to break.